src/design-system/tokens.scss

```diff
@@ -39,4 +44,13 @@
   --x-size-font-input-group-default: var(--x-size-font-base-m);
   --x-size-gap-input-group-default: var(--x-size-base-05);
   --x-size-height-input-group-default: var(--x-size-base-08);
+
```

Comment:
// Result
  --x-size-gap-result-default: var(--x-size-base-02);

Reply:
This is needed because the distance between the image and the title is 8px instead of 4px.

Reply:
This also changes in mobile to 2px.

Reply:
Yep, fixed 👌